src/commands/service/create.test.ts

```diff
@@ -1,10 +1,11 @@
 import fs from "fs";
-import os from "os";
 import path from "path";
 import { promisify } from "util";
 import uuid from "uuid/v4";
 import { Bedrock } from "../../config";
+import * as config from "../../config";
 import { checkoutCommitPushCreatePRLink } from "../../lib/gitutils";
+import { createTempDir, removeDir } from "../../lib/ioUtil";
```

Comment:
`createTempDir` is not implemented in this PR, but `createDir` is a preferable name and caller can decide how the dir being used.

Reply:
`createTempDir` function already exist (added in previous PR)